src/redux/store.tsx

```diff
@@ -1,17 +1,11 @@
 import { configureStore } from '@reduxjs/toolkit';
+
 import authReducer from './slice/authSlice';
 
 const store = configureStore({
   reducer: {
-
-    auth: authReducer
-  }
-})
-
-export type RootState = ReturnType<typeof store.getState>
-
     auth: authReducer,
   },
```

Comment:
This is an update provided by Elisa on slack but I did not find it on github when I pulled